django/db/backends/sqlite3/introspection.py

```diff
@@ -36,10 +36,12 @@ class FlexibleFieldLookupDict:
         'int': 'IntegerField',
         'integer': 'IntegerField',
         'bigint': 'BigIntegerField',
+        'unsigned integer': 'PositiveIntegerField',
         'integer unsigned': 'PositiveIntegerField',
         'bigint unsigned': 'PositiveBigIntegerField',
         'decimal': 'DecimalField',
         'real': 'FloatField',
+        'double': 'FloatField',
```

Comment:
I don't think we should add the `DOUBLE`, see [comment](https://code.djangoproject.com/ticket/32672#comment:5).